llvm/test/tools/sycl-post-link/spec-constants/SYCL-2020-zeroinitializer.ll

```diff
@@ -0,0 +1,104 @@
+; RUN: sycl-post-link --ir-output-only --spec-const=rt %s -S -o - | FileCheck %s
+;
+; This test is intended to check that SpecConstantsPass is able to handle the
+; situation where specialization constants have zeroinitializer in LLVM IR
+;
+; CHECK: call i32 @_Z20__spirv_SpecConstantii(i32 [[#SCID0:]], i32 0)
```

Comment:
please move the checks next to the source IR

Reply:
Done in 1a9927894d4ad79151f89ff742454c848849af68